.github/workflows/build-publish.yml

```diff
@@ -0,0 +1,88 @@
+name: Build
+on: 
+  push:
+    branches:
+      - main
+    tags:
+    - 'v*'
```

Comment:
is this indentation correct? I would have expected it to be on the same level as `branches` above

Reply:
The indentation is correct but I will make it consisted